KnockKnock-iOS/Scenes/Cells/ChallengeCell.swift

```diff
@@ -7,32 +7,108 @@
 
 import UIKit
 
-final class ChallengeCell: BaseTableViewCell {
+final class ChallengeCell: BaseCollectionViewCell {
   
   // MARK: - Constants
   
   private enum Metric {
     static let challengeHeight = 245.f
+
+    static let newChallengeLabelTopMargin = 11.f
+    static let newChallengeLabelLeadingMargin = 10.f
+    static let newChallengeLabelWidth = 40.f
+    static let newChallengeLabelHeight = 20.f
+
+    static let hotChallengeLabelTopMargin = 11.f
+    static let hotChallengeLabelLeadingMargin = 5.f
+    static let hotChallengeLabelWidth = 40.f
+    static let hotChallengeLabelHeight = 20.f
+
+    static let participantViewTopMargin = -15.f
+    static let participantViewTrailingMargin = -77.f
+    static let participantViewHeight = 40.f
+
+    static let participantImageViewTopMargin = 15.f
+
+    static let participantLabelViewTopMargin = 15.f
+    static let participantLabelViewLeadingMargin = 5.f
+
+    static let seperatorLineViewHeight = 1.f
+    static let seperatorLineViewTopMargin = 10.f
+
+    static let titleLabelTopMargin = 10.f
+
+    static let contentsLabelTopMargin = 5.f
   }
  
   // MARK: - UI
   
   let challengeImageView = UIImageView().then {
     $0.translatesAutoresizingMaskIntoConstraints = false
+    $0.layer.cornerRadius = 5
     $0.image = UIImage(named: "challenge")
   }
```

Comment:
```suggestion
private  let challengeImageView = UIImageView().then {
    $0.translatesAutoresizingMaskIntoConstraints = false
    $0.layer.cornerRadius = 5
    $0.image = UIImage(named: "challenge")
  }
```
다음과 같이 접근지정자를 선언해주시는게 좋아보입니다!!

Reply:
ChallengeCell 에 있는 모든 UI 프로퍼티는 캡슐화 부탁드릴게요!!